tests/testthat/test-Data-validity.R

```diff
@@ -6,90 +6,62 @@ test_that("validate_subjects returns TRUE for valid object", {
 })
 
 test_that("validate_subjects returns error message for non-valid object", {
-  expect_error(.GeneralData(ID = 1:3, cohort = 3:5, nObs = 4), regexp = ".+")
+  object <- expect_silent(.GeneralData(ID = 1:3, cohort = 3:5, nObs = 3L))
+  object@nObs <- 4L
+  expect_equal(
+    object = validate_subjects(object),
+    expected = c(
```

Comment:
generally for these expect functions - no need to give explicit argument names. can just say `expect_*(result, expected)` without `object =` etc.